app/components/Button/Button.jsx

```diff
@@ -10,25 +10,32 @@ type Props = {
   renderIcon: ?Function,
   primary: ?boolean,
   type: ?string,
+  displayButtonIcon?: boolean,
```

Comment:
By excluding the prop `renderIcon` Im pretty sure this component will render a button without the icon so displayButtonIcon might be unnecessary unless Im mistaken

Reply:
I thought so too, however as far as I can tell it will render an empty div thats still takes up space and skews the text in the button: 
```
  renderIcon = () => {
    const { renderIcon } = this.props

    if (renderIcon) {
      return (
        <span className={classNames(styles.icon, this.getIconStyle())}>
          {renderIcon()}
        </span>
      )
    }
    return <div className={styles.icon} />
  }
```